a11y.config.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+  storybookBuildDir: '.out',
+  components: [],
+  pa11y: {
+    includeNotices: true,
+    includeWarnings: true,
+    runners: ['axe'],
+  },
+  // A11y linting is done on a component-by-component
+  // basis, which results in the linter reporting some errors that
+  // should be ignored. These codes and descriptions allow for those
+  // errors to be targeted specifically.
+  ignore: {
+    codes: ['landmark-one-main', 'page-has-heading-one'],
+    descriptions: ['Ensures all page content is contained by landmarks'],
+  },
+  // List of storybook component IDs defined and used in this project.
+  components: [
```

Comment:
Well, I tried my very best to load this up dynamically, but I hit big road blocks on every path. I understand that maintaining a list of component ids is not ideal, but I'm afraid it's the only way we can reliably do identify the components without upgrading to the latest storybook beta.

Once the new storybook version is released, we should be able to use a new export command from the clip. Until then, this is as good as it gets :(

Reply:
There is one more possibility that I didn't have time to poke at... we could try running the script with babel-node or something along those lines, and we could import the story js files directly. If ya'll want me to give that a shot, lmk!